sdk/keyvault/keyvault-keys/src/cryptographyClientModels.ts

```diff
@@ -17,11 +16,44 @@ export {
   KeyCurveName,
   KnownKeyCurveNames,
   EncryptionAlgorithm,
-  KnownEncryptionAlgorithms,
   SignatureAlgorithm,
   KnownSignatureAlgorithms,
 };
 
+/** Known values of {@link EncryptionAlgorithm} that the service accepts. */
+export enum KnownEncryptionAlgorithms {
```

Comment:
Oh yeah, I forgot I did this. At some point the swagger removed all these comments and I got tired of adding them back in every time I regenerate!

Reply:
Can you prevent the generated enum from ending up in source then? Or is the idea with minimization is that apps should use webpack or something to remove unused code?

Reply:
We already GA'd with this enum in-place, it was just re-exported directly from generated code